src/index.css

```diff
@@ -14,6 +14,59 @@ body {
   margin-top: 15px;
 }
 
+.album-container {
+  width: 300px;
+  height: 300px;
+}
```

Comment:
instead of doing this, we can add these properties instead so that this class gets resized automatically depending on the screen size:

```
max-width: 600px;
aspect-ratio: 1 / 1;
```

Reply:
Check!